### PR TITLE
Added_function_to_manage_SDDC_host

### PR DESCRIPTION
**Pull request by jain-prerna**

### CODE REVIEW

File: src/saltext/vmware/modules/vmc_sddc_host.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    params = vmc_request._filter_kwargs(allowed_kwargs=["action"], action=action)
+    request_data = vmc_request.create_payload_for_request(vmc_templates.manage_sddc_host, req_data)
+    return vmc_request.call_api(
+        method=vmc_constants.POST_REQUEST_METHOD,
+        url=api_url,
+        refresh_key=refresh_key,
+        authorization_host=authorization_host,
+        description="vmc_sddc_host.manage",
+        data=request_data,
+        params=params,
```
Comment (waynew) [inline src/saltext/vmware/modules/vmc_sddc_host.py:144]: **suggestion** these parameters are not being tested (i.e. they can be replaced with `None` and all tests still pass)

Comment (waynew) [reply src/saltext/vmware/modules/vmc_sddc_host.py:144]: What I mean is that 

```
params = vmc_request._filter_kwargs(allowed_kwargs=["action"], action=action)
```

Could be replaced with

```
params = None
```

And there are no tests that would detect this problem.

Comment (jain-prerna) [reply src/saltext/vmware/modules/vmc_sddc_host.py:144]: Added the required test.

**Comment by jain-prerna** [comment]
> Looks like some tests are missing here, otherwise I didn't notice anything that needs to be addressed

I have added it, please do check now.

**Review by waynew — APPROVED** [review]
:+1:

**Pull request merged**